.ko.yaml

```diff
@@ -15,3 +15,49 @@
 
 # We need a shell for a lot of redirection/piping to work
 defaultBaseImage: gcr.io/distroless/base:debug-nonroot
+
+builds:
+- id: cosign
+  dir: .
+  main: ./cmd/cosign
+  env:
+  - CGO_ENABLED=0
+  flags:
+  - -trimpath
+  - -tags
+  - "{{ .Env.GIT_HASH }}"
+  - -tags
+  - "{{ .Env.GIT_VERSION }}"
+  ldflags:
+  - -extldflags "-static"
```

Comment:
can't we add them to the LDFLAGS environment variable as [the ossf/scorecard](https://github.com/ossf/scorecard/blob/main/scripts/version-ldflags#L27) project did? @cpanato

Reply:
we can, but here define that as well, not sure what is the best